MapboxCoreNavigation/CLLocation.swift

```diff
@@ -3,6 +3,12 @@ import CoreLocation
 
 extension CLLocation {
     
+    var isQualified: Bool {
+        return
+            0...100 ~= horizontalAccuracy &&
```

Comment:
If the wifi is off and the user is in an urban environment, will this ever be below 100m? Should we start higher and filter down slowly?